kubernetes/config-maps/nginx-proxy/configs/http.conf.j2

```diff
@@ -20,15 +20,19 @@ upstream {{ current_golem_messages_version }} {
      server nginx-proxy.default.svc.cluster.local;
 }
 
+{% if previous_golem_messages_version is not none and previous_golem_messages_version is defined %}
 upstream {{ previous_golem_messages_version }} {
      server {{ concent_versions[previous_concent_version].cluster_address }}:443;
 }
```

Comment:
This should have a comment explaining why this is enabled only when `previous_golem_messages_version` is non-empty.